### PR TITLE
Create readme.md

### PR DESCRIPTION
**Pull request by windreis**

### CODE REVIEW

**Comment by NelsonBN** [comment]
Nice PR. Let's go...

File: readme.md [hunk 1]
```diff
@@ -0,0 +1 @@
+# hello world!
```
Comment (NelsonBN) [inline readme.md:1]: Can you add more details

**Review by NelsonBN — CHANGES_REQUESTED** [review]
Please look my inline comments

File: readme.md (hunk 1 above)
Comment (windreis) [reply readme.md:1]: You are right, aI added more info...

**Review by NelsonBN — APPROVED** [review]
LGTM now

**Pull request merged**